java/core/src/main/scala/io/pdal/PointView.scala

```diff
@@ -0,0 +1,77 @@
+package io.pdal
+
+import java.nio.ByteBuffer
+
+class PointView extends Native {
+  def getPackedPointWithMetadata(idx: Long, metadata: String, schema: String): PackedPoints =
+    getPackedPoint(idx).copy(metadata = metadata, schema = schema)
+  def getPackedPoint(idx: Long): PackedPoints = getPackedPoint(idx, layout.dimTypes())
```

Comment:
I'm a bit confused about what the difference between a PointView and PackedPoints is. Isn't this "PackedPoints" going to just wrap a single point? The fact that you get a class named `PackedPoints` (or potentially PointCloud) is strange here because you're getting a single point.